compiler-core/src/error.rs

```diff
@@ -321,6 +321,9 @@ file_names.iter().map(|x| x.as_str()).join(", "))]
 
     #[error("Failed to decrypt data")]
     FailedToDecrypt { detail: String },
+
+    #[error("Failed to decrypt API key")]
+    FailedToDecryptApiKey { detail: String },
```

Comment:
Could be any API key! FailedToDecryptLocalHexApiKey or such? The error attribute needs to say this info also.